solidity/contracts/TokenStaking.sol

```diff
@@ -20,7 +20,12 @@ contract TokenStaking is StakeDelegatable {
 
     // Minimum amount of KEEP that allows sMPC cluster client to participate in
     // the Keep network. Expressed as number with 18-decimal places.
-    uint256 public minimumStake = 200000 * 1e18;
+    // Initial minimum stake is higher than the final and lowered periodically based
+    // on the amount of steps and the length of the minimum stake schedule in blocks.
+    uint256 public minimumStakeScheduleStart;
+    uint256 public minimumStakeSchedule = 86400 * 365 * 2; // 2 years in seconds (seconds per day * days in a year * years)
+    uint256 public minimumStakeSteps = 10;
+    uint256 public minimumStakeBase = 10000 * 1e18;
```

Comment:
The minimum stake schedule could be packed into a single storage field with the following:
```
uint64 public minimumStakeScheduleStart;
uint64 public minimumStakeSchedule = 86400 * 365 * 2;
uint32 public minimumStakeSteps = 10;
uint96 public minimumStakeBase = 10000 * 1e18;
```

Reply:
since we're going to store it only once and then it's read only, extra gas for unpacking probably not worth it?

Reply:
With read-only we can still save 2400 gas from the `SREAD`s at the cost of a couple dozen gas for unpacking. Not really critical but can be nice.